contracts/models/Reaction.sol

```diff
@@ -66,5 +66,6 @@ struct Reaction {
   uint postId;
   uint commentId;
   uint8 reactionType;
+  uint reactionId;
```

Comment:
We should probably not save `reactionId` in multiple places, this use is unnecessary